gidgethub/app.py

```diff
@@ -0,0 +1,48 @@
+"""Support for GitHub Actions."""
+from typing import Any
+
+import time
+import jwt
+
+from gidgethub.abc import GitHubAPI
+
+
+def _get_jwt(*, app_id: str, private_key: str) -> str:
+    """Construct the JWT (JSON Web Token), used for GitHub App authentication."""
+    time_int = int(time.time())
+    payload = {
+        "iat": time_int,
+        "exp": time_int + (10 * 60),
+        "iss": app_id,
+    }
+    encoded = jwt.encode(payload, private_key, algorithm="RS256")
+    bearer_token = encoded.decode("utf-8")
+
+    return bearer_token
+
+
+async def get_installation_access_token(
+    gh: GitHubAPI, *, installation_id: str, app_id: str, private_key: str
+) -> Any:
```

Comment:
Since this is calling a specific endpoint that has an expected value, maybe we should type this as `Dict[str, str]`?

Reply:
This returns whatever `gh.post` returns, which is `Any`. When I tried to change this to `Dict[str, str]`, it complained because it doesn't match `gh.post`'s return type.

Reply:
Yeah, you probably would have to [cast](https://docs.python.org/3/library/typing.html#typing.cast) it.